modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/dependencies/walkthrough_configuration_onePage.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Let's name the file `walkthrough_configuration_single_page.json` I think single will better distinguish it from multi_page. We may want to also change the other .json file to `...multi_page` to complete the refactor.